pkg/kapis/gitops/v1alpha1/argocd/route.go

```diff
@@ -104,22 +67,6 @@ func RegisterRoutes(service *restful.WebService, options *common.Options, argoOp
 		Doc("Sync a particular application manually").
 		Returns(http.StatusOK, api.StatusOK, v1alpha1.Application{}))
 
-	service.Route(service.DELETE("/namespaces/{namespace}/applications/{application}").
-		To(handler.delApplication).
-		Param(common.NamespacePathParameter).
-		Param(pathParameterApplication).
-		Param(cascadeQueryParam).
-		Doc("Delete a particular application").
-		Returns(http.StatusOK, api.StatusOK, v1alpha1.Application{}))
-
-	service.Route(service.PUT("/namespaces/{namespace}/applications/{application}").
-		To(handler.updateApplication).
-		Param(common.NamespacePathParameter).
-		Param(pathParameterApplication).
-		Reads(v1alpha1.Application{}).
-		Doc("Update a particular application").
-		Returns(http.StatusOK, api.StatusOK, v1alpha1.Application{}))
-
 	service.Route(service.GET("/clusters").
```

Comment:
Is `/clusters` api available for fluxcd?

Reply:
Yes. I think you're right. fluxcd should also has `/clusters` API. I'll fix this.